### DISCUSSION TITLE
Drift detected: We need to reset the PostgreSQL database

### DISCUSSION THREAD

**Discussion by Gmulti** [prisma/prisma#5735] [Q&A]
Hello,

I'm currently updating my Prisma with the new migration system that came out in version 2.13.

I am following the recommendations of your documentation: https://www.prisma.io/docs/guides/prisma-guides/add-prisma-migrate-to-a-project#baseline-your-production-environment

I have migrated well in development only at the production stage and even with an "--applied" nothing is done about it... he wants to empty my production database which is not acceptable.

Would you have a solution to this problem?
Why does he see my database that is not "sync" when I have also made an introspect...

**Comment by ryands17** [comment]
@Gmulti 👋 

Could you share the schema and the output you get after running the resolve command? It would also be great if you could share the migration files created so that we can debug this :)

**Reply by ryands17** [reply]
@Gmulti 👋 

Was the above issue solved for you? If not, could you share the CLI command used and the output?

**Comment by Gmulti** [comment]
Hi @ryands17 

CLI command : `prisma migrate dev --preview-feature`

<img width="255" alt="Capture d’écran 2021-02-23 à 23 00 17" src="https://user-images.githubusercontent.com/5682535/108913988-07b66f80-762b-11eb-92a1-c7e5fd3e7fa4.png">

Concerning the schema and the migration file, would it be possible to send it to you privately?

**Reply by Jolg42** [reply]
Yes! send that to schemas@prisma.io with a link to this discussion 💯

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
